addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -698,26 +698,24 @@ function factory(dependencies) {
             } else {
                 const authorName = author.nameOrDisplayName;
                 if (channel.channel_type === 'channel') {
-                    // hack: notification template does not support OWL components,
-                    // so we simply use their template to make HTML as if it comes
-                    // from component
-                    const channelIcon = this.env.qweb.renderToString('mail.ThreadIcon', {
-                        env: this.env,
-                        thread: channel,
-                    });
+                    const channelIcon = channel.public === 'private' ? '<i class="fa fa-lock"/> '
+                                                                     : '<i class="fa fa-hashtag"/> ';
                     const channelName = owl.utils.escape(channel.displayName);
                     const channelNameWithIcon = channelIcon + channelName;
                     notificationTitle = _.str.sprintf(
-                        this.env._t("%s from %s"),
+                        this.env._t("<b>%s from %s</b>"),
                         owl.utils.escape(authorName),
                         channelNameWithIcon
                     );
                 } else {
-                    notificationTitle = owl.utils.escape(authorName);
+                    notificationTitle = _.str.sprintf(
+                        this.env._t("<b>%s</b>"),
+                        owl.utils.escape(authorName)
+                    );
                 }
             }
-            const notificationContent = message.prettyBody.substr(0, PREVIEW_MSG_MAX_SIZE);
-            this.env.services['bus_service'].sendNotification(notificationTitle, notificationContent);
+            const notificationContent = notificationTitle + message.prettyBody.substr(0, PREVIEW_MSG_MAX_SIZE);
```

Comment:
I know this was already like that before, but doing `substr` on something that contains HTML can lead to broken tags, this seems like a bad idea. Would probably be best with something similar to `inlineLastMessageBody` from thread preview.